kafka/channel/config/README.md

```diff
@@ -0,0 +1,92 @@
+# Apache Kafka Channels
+
+Kafka channels are those backed by [Apache Kafka](http://kafka.apache.org/)
+topics.
+
+## Deployment steps
+
+1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
```

Comment:
This points to setting up the dev stack (including things like ko, which is probably not necessary just to install kafka channels?)

Reply:
nm, I see ko being used below. However, we should probably have a kubectl version and released binaries for people so that they can just install a release version.

Reply:
sure, let me do that in a different PR - this was mainly just porting it over and test it